app/models/edition.rb

```diff
@@ -51,6 +51,10 @@ class Edition < ApplicationRecord
   validates :body, presence: true, if: :body_required?, length: { maximum: 16777215 }
   validates :summary, presence: true, if: :summary_required?, length: { maximum: 65535 }
   validates :first_published_at, recent_date: true, allow_blank: true
+  validates :first_published_at, previously_published: true
+  validates_each :first_published_at do |record, attr, value|
+    record.errors.add(attr, "can't be set to a future date") if Time.zone.now < value
```

Comment:
This looks good.

I was worried that this might catch people out as there are a bunch of editions with future publishing dates:

```
irb(main):005:0> Edition.where("first_published_at > ?", Time.zone.now).count
=> 38
```

However they are all superseded:

```
irb(main):006:0> Edition.where("first_published_at > ?", Time.zone.now).distinct.pluck(:state)
=> ["superseded"]
```

So I think we should probably go for this.

If it does transpire though that publishers have a valid reason to set first_published_at date into the future then we may have to add a conditional to this which only applies this check if `political` is true (which, as I understand, is the attribute that associates this record with a government)